tests/unittests/test_dispatcher.py

```diff
@@ -618,3 +620,41 @@ async def test_dispatcher_load_azfunc_in_init(self):
                 1
             )
         self.assertIn("azure.functions", sys.modules)
+
+
+class TestConfigurableFileName(testutils.AsyncTestCase):
+
+    def setUp(self):
```

Comment:
We might also need to add an e2e tests with the new file name. You can create a new test function app with a new file name in tests/endtoend/http_functions/http_functions_stein/ and write a test in test_http_functions.py